web-app/webpack.config.js

```diff
@@ -42,6 +62,40 @@ module.exports = {
         'process.browser': true,
         'process.env.NODE_ENV': JSON.stringify(mode)
       }),
+      new MiniCssExtractPlugin({
+        filename: '[hash]/[name].css',
+        chunkFilename: '[hash]/[id].css',
+      }),
+      new PurifyCSSPlugin({
+        paths: glob.sync([
+          path.join(__dirname, 'src/**/*.html'),
+          path.join(__dirname, 'src/**/*.svelte')
+        ]),
+        minimize: !dev
+      }),
+      new WebpackPwaManifest({
+        // Config
+        filename: "manifest.json",
+        inject: false,
+        publicPath: '/client',
+
+        // Manifest properties
+        "background_color": "#50AF4F",
+        "theme_color": "#EF753D",
+        "name": "Pub Golf",
+        "short_name": "PubG",
+        "display": "minimal-ui",
+        "start_url": "/nyc-2019/auth",
+
+        // Dynamic image generation
+        icons: [
+          {
+            src: path.resolve('src/assets/images/social-beer--green.png'),
+            sizes: [96, 128, 192, 256, 384, 512, 1024],
```

Comment:
Do you still need
- `web-app/static/logo-192.png → web-app/src/assets/images/logo-192.png	` and
- `web-app/static/logo-512.png → web-app/src/assets/images/logo-512.png`

in addition to this?

Reply:
Nope, good catch